docs/versioned_docs/version-2.x/domain.mdx

```diff
@@ -54,26 +54,30 @@ intents:
       - age
 ```
 
-This means that excluded entities for those intents will be unfeaturized and therefore
+Excluded entities for those intents will be unfeaturized and therefore
 will not impact the next action predictions. This is useful when you have
 an intent where you don't care about the entities being picked up.
 
 If you list your intents without this parameter, the entities will be
 featurized as normal.
 
 :::note
-If you really want these entities not to influence action prediction we
-suggest you set the [`influence_conversation: false`](./domain.mdx#slots-and-conversation-behavior)
+If you want these entities not to influence action prediction,
+set the [`influence_conversation: false`](./domain.mdx#slots-and-conversation-behavior)
 parameter for slots with the same name.
 
 :::
 
 ## Entities
 
-The `entities` section lists all entities
+The `entities` section lists all entities that can be
 extracted by any [entity extractor](./components.mdx) in your
 NLU pipeline.
 
+:::caution Changed in 2.1
```

Comment:
this example makes me wish for a simpler way to signify the change... smth like "If you are using the feature Entity Roles and Groups introduced in 2.1..."

because the entire entities section at this point is one sentence and then a big yellow banner...

Reply:
I see your point. I think this can be fixed by rewording the section a bit by doing:
- a default example (without entities roles and groups)
- an example with entities roles and groups

I think we could reduce the box here to a minimal sentence, and describe the bigger changes needed below. Would that work for you?

Reply:
yep I meant to comment back that I saw that somewhere else, where we have a minimal sentence in the box and had the current behavior below in a normal space. Let's do that here